### PR TITLE
SDK update for 2023-04-01 GA version

### PR DESCRIPTION
**Pull request by Joeleniqs**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/communication/Azure.ResourceManager.Communication/tests/Infrastructure/CommunicationManagementClientLiveTestBase.cs [hunk 1]
```diff
@@ -96,43 +96,43 @@ internal async Task<SenderUsernameResource> CreateDefaultSenderUsernameResource(
...
-                LastName = lastName,
-                Notes = notes
-            };
-
-            ArmOperation<SuppressionListAddressResource> suppressionListAddressOp =
-                await suppressionList.GetSuppressionListAddressResources().CreateOrUpdateAsync(WaitUntil.Completed, id, data);
-            SuppressionListAddressResource suppressionListAddress = suppressionListAddressOp.Value;
-            return suppressionListAddress;
-        }
+        //internal async Task<SuppressionListResource> CreateDefaultSuppressionListResource(CommunicationDomainResource domain, string listName)
```
Comment (live1206) [inline sdk/communication/Azure.ResourceManager.Communication/tests/Infrastructure/CommunicationManagementClientLiveTestBase.cs:99]: why is this commented out? if not needed, can we just remove it?

File: sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs [hunk 2]
```diff
@@ -15,266 +15,266 @@
...
-
-            // // todo: follow up on this issue. getting item after being deleted should return 404 not found instead of 500.
-            // collection = suppressionList.GetSuppressionListAddressResources();
-            // exists = await collection.ExistsAsync(addressResource.Id.Name);
-            // Assert.IsFalse(exists);
-
-            await suppressionList.DeleteAsync(WaitUntil.Started);
-        }
-    }
+    //public class SuppressionListTests : CommunicationManagementClientLiveTestBase
```
Comment (live1206) [inline sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs:18]: why is the whole file commented out?

File: sdk/communication/Azure.ResourceManager.Communication/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.2.0 (2024-02-19)
+
+Stable release of the 2023-04-01 api version, specifically for the support of Managed Identity in Azure Communication services.
```
Comment (live1206) [inline sdk/communication/Azure.ResourceManager.Communication/CHANGELOG.md:5]: please put this informatino into `Features added` section.

**Review by live1206 — COMMENTED** [review]
Please resolve the merge conflicts.

File: sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs (hunk 2 above)
Comment (Joeleniqs) [reply sdk/communication/Azure.ResourceManager.Communication/tests/ScenarioTests/SuppressionListTests.cs:18]: They are scenario tests written for a preview SDK off this API version. The tests are still relevant, not just for this Version.

File: sdk/communication/Azure.ResourceManager.Communication/tests/Infrastructure/CommunicationManagementClientLiveTestBase.cs (hunk 1 above)
Comment (Joeleniqs) [reply sdk/communication/Azure.ResourceManager.Communication/tests/Infrastructure/CommunicationManagementClientLiveTestBase.cs:99]: We would be removing work that someone did and has need of for a future version. They just happened to release a preview sdk before this stable release

Comment (live1206) [reply sdk/communication/Azure.ResourceManager.Communication/tests/Infrastructure/CommunicationManagementClientLiveTestBase.cs:99]: The test can be recovered from the git history as well.
Either way is fine.

**Review by live1206 — APPROVED** [review]

**Pull request merged**